servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -1467,7 +1467,6 @@ bool LightStorage::reflection_probe_instance_begin_render(RID p_instance, RID p_
 		fb.push_back(atlas->depth_buffer);
 		atlas->depth_fb = RD::get_singleton()->framebuffer_create(fb);
 
-		atlas->render_buffers->cleanup();
```

Comment:
is this related?

Reply:
No, just ran into it as I was debugging, it actually calls `cleanup` when it calls `configure` in the next line so it's double.